README.md

```diff
@@ -1,2 +1,151 @@
-# Leader-board
-a simple leader board which works with API
+<a name="readme-top"></a>
+
+<div>
+
+  <h1><b>Leader board</b></h1><br/><br/>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Usage](#usage)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# 📖 Leader board <a name="about-project"></a>
+
+**Leader board**
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<summary>Client</summary>
+<ul>
+  <li><a href="https://html.com/html5/">HTML5</a></li>
+  <li><a href="https://developer.mozilla.org/en-US/docs/Learn/CSS/First_steps/What_is_CSS">CSS</a></li>
+  <li><a href="https://www.javascript.com/">JavaScript</a></li>
+</ul>
+
+### Key Features <a name="key-features"></a>
+
+- **Add and remove tasks**
+
+- **JavaScript Classes**
+
+  - The tasks are managed and manipulated using classes and helper functions within those classes.
+
+- **Store and retrieve data**
+
+  - The displayed tasks are stored and retrieved from the local storage of the browser.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- A browser (e.g. Firefox)
+- An IDE (e.g. [Vususal Studio Code](https://code.visualstudio.com/download))
+- [npm](https://nodejs.org/en/)
+- Terminal
+
+### Setup
+
+To clone this repository to your desired folder, follow the steps below:
+
+**Using the command line**
+
+- Use the following commands
+
+```sh
+  cd my-folder
+  git clone git@github.com:mohisa302/Leader-board.git
+```
+
+**Using GitHub Desktop app**
+
+- Follow the steps below
+
+  - Visit this link "git@github.com:mohisa302/Leader-board.git"
+  - Click the green button labelled "code"
+  - Select the "Open with GitHub Desktop" option
+  - After the GitHub Desktop add opens, click the "clone repo" button
+
+### Usage
+
+To run the project, execute the following command:
+
+**Click the "Go Live" button on your IDE(VS Code in this case)**
```

Comment:
- **[OPTIONAL]** Actually, I didn't use "Go live" please 🙏🏽 actualize your instructions, To have a professional README file. I ran:
- npm install
- npm start

Reply:
done